src/core/common/settings.hpp

```diff
@@ -802,6 +804,34 @@ class Settings : public SettingsBase, private NonCopyable
      */
     void DeleteOperationalDataset(MeshCoP::Dataset::Type aType);
 
+#if OPENTHREAD_CONFIG_BLE_TCAT_ENABLE
+    /**
+     * Stores the Tcat Commissioner certificate.
+     *
+     * @param[in]  aCert            The DER-encoded X509 end-entity certificate to store.
+     * @param[in]  aCertLen         Certificate length.
+     *
+     */
+    void SaveTcatCommissionerCertificate(uint8_t *aCert, uint16_t aCertLen);
+
+    /**
+     * Reads the Tcat Commissioner certificate.
+     *
+     * @param[out]  aCert            Buffer to store the DER-encoded X509 end-entity certificate of the TCAT
+     * Commissioner
+     * @param[in,out]  aCertLen      On input, the max size of @p aCert. On output, the length of the DER encoded peer
+     * certificate.
+     *
+     * @retval kErrorNone             Successfully read the Dataset.
+     * @retval kErrorNotFound         No corresponding value in the setting store.
+     * @retval kErrorNoBufs          Buffer has not enough space to store the data.
+     */
+    Error GetTcatCommissionerCertificate(uint8_t *aCert, uint16_t &aCertLen)
```

Comment:
Rename to  `ReadTcat...`? 

For consistency with all other `Settings` methods which `Save` and `Read`

Reply:
fixed.